src/userscores/qml/MuseScore/UserScores/internal/ChooseTemplatePage.qml

```diff
@@ -53,6 +58,10 @@ Item {
             Layout.fillHeight: true
             Layout.preferredWidth: parent.width / 4
 
+            navigation.section: root.navigationSection
+            navigation.name: "Category"
+            navigation.order: 2
```

Comment:
Why does it start from 2 and not from 1?

Reply:
we have one panel before 

![image](https://user-images.githubusercontent.com/3818029/117163282-cc22e900-adc3-11eb-9b69-80922ed21d8f.png)